Makefile

```diff
@@ -26,7 +26,7 @@ endif
 
 # Temporary
 OPENSCAP_DEFAULT_IMAGE_TAG=1.3.4
-RELATED_IMAGE_OPENSCAP_TAG?=$(OPENSCAP_DEFAULT_IMAGE_TAG)
+RELATED_IMAGE_OPENSCAP_TAG?=latest
```

Comment:
I think we should stop using `OPENSCAP_DEFAULT_IMAGE_TAG` and just stick to the variable `RELATED_IMAGE_OPENSCAP_TAG`.

Reply:
We are using 1.3.4 at https://github.com/openshift/compliance-operator/blob/50cbfab8678e1164fdbea713c8f57303612f2311/deploy/operator.yaml#L48, should we change here as well then?